mpas_analysis/config.default

```diff
@@ -1244,6 +1244,12 @@ variables = [{'name': 'temperature',
               'units': 'kg m$^{-3}$',
               'mpas': 'timeMonthly_avg_potentialDensity'}]
 
+# The minimum and maximum depth over which fields are averaged, default is
+# to take these values from the geojson feature's zmin and zmax properties.
+# Add these to a custom config file to override the defaults.
+# zmin = -1000
+# zmax = -400
+
```

Comment:
Although unrelated to this PR, I think it would be nice to go down to the WOCE transects section, here:
```
# Horizontal bounds of the plot (in km), or an empty list for automatic bounds
# The bounds are a 2-element list of the minimum and maximum distance along the
# transect
horizontalBounds = {'WOCE_A21': [],
                    'WOCE_A23': [],
                    'WOCE_A12': []}
```
and add a comment that says where these transects are in words like we used to have (A21=Drake Passage; A23=South Atlantic; A12=Prime Meridian), to help the user adding WOCE transects to their customized config file.

Reply:
I'll fix this and address #599 in a separate PR, since that one can focus on WOCE.